frontend/src/api/authRequest.test.tsx

```diff
@@ -21,5 +22,23 @@ describe('use of AuthApiClient', () => {
 
 })
 
+describe('use of TokenRefresh', () => {
```

Comment:
Does this test really test `TokenRefresh`?

Reply:
corrected